pkg/plugins/resources/postgres/store.go

```diff
@@ -127,6 +127,20 @@ func (r *postgresResourceStore) Delete(_ context.Context, resource model.Resourc
 	return nil
 }
 
+func (r *postgresResourceStore) DeleteMany(ctx context.Context, fs ...store.DeleteManyOptionsFunc) error {
```

Comment:
I don't think it's a good idea to remove resources without taking into account their type